tmol/score/omega/omega_energy_term.py

```diff
@@ -0,0 +1,88 @@
+import torch
+
+from ..atom_type_dependent_term import AtomTypeDependentTerm
+from ..bond_dependent_term import BondDependentTerm
+
+from tmol.database import ParameterDatabase
+from tmol.score.common.stack_condense import tile_subset_indices
+from tmol.score.omega.params import OmegaGlobalParams
+from tmol.score.omega.omega_whole_pose_module import OmegaWholePoseScoringModule
+
+from tmol.chemical.restypes import RefinedResidueType
+from tmol.pose.packed_block_types import PackedBlockTypes
+from tmol.pose.pose_stack import PoseStack
+
+
+class OmegaEnergyTerm(AtomTypeDependentTerm, BondDependentTerm):
+    tile_size: int = 32
+
+    def __init__(self, param_db: ParameterDatabase, device: torch.device):
+        super(OmegaEnergyTerm, self).__init__(param_db=param_db, device=device)
+
+        self.global_params = OmegaGlobalParams.from_database(
+            param_db.scoring.omega, device
+        )
+        self.tile_size = OmegaEnergyTerm.tile_size
+
+    @classmethod
+    def score_types(cls):
+        import tmol.score.terms.omega_creator
+
+        return tmol.score.terms.omega_creator.OmegaTermCreator.score_types()
+
+    def n_bodies(self):
+        return 2
```

Comment:
Technically the calculation involves 2 residues, but it is O(n) on the number of blocks. What should this be?